README.md

```diff
@@ -32,8 +34,7 @@ support privileged extension, root installation, or local repositories.
 
 ## Building and Installing
 
-Android SDK path should be specified either using `ANDROID_HOME` environment variable or using `sdk.dir` property in
-`local.properties`.
+Specify your Android SDK path either using the `ANDROID_HOME` environment variable, or filling out the `sdk.dir` property in `local.properties`.
```

Comment:
```suggestion
To join the development, specify your Android SDK path either using the `ANDROID_HOME` environment variable, or by filling out the `sdk.dir` property in `local.properties`.
```
How about this, the idea is to allow those that don't need it to discard the info without being scared of it.